diff -Naur liquidwar-5.6.0/Makefile.in liquidwar-5.6.0-/Makefile.in
--- liquidwar-5.6.0/Makefile.in	Thu Dec 18 15:02:28 2003
+++ liquidwar-5.6.0-/Makefile.in	Sat Dec 20 20:36:12 2003
@@ -157,8 +157,8 @@
 
 install_desktop:
 	@echo "Installing desktop file in $(DESKTOPDIR)."
-	@install -d $(DESKTOPDIR)
-	@install -m 0644 misc/liquidwar.desktop $(DESKTOPDIR)
+	@install -d $(DESTDIR)$(DESKTOPDIR)
+	@install -m 0644 misc/liquidwar.desktop $(DESTDIR)$(DESKTOPDIR)
 
 install_gpl:
 	@echo "Installing GPL in $(DOCDIR)."
